test-support/build.gradle

```diff
@@ -0,0 +1,67 @@
+import org.gradle.util.GradleVersion
+
+plugins {
+    id 'java-library'
+    id 'maven-publish'
+}
+
+version = '0.18-SNAPSHOT'
+
+dependencies {
+    api project(':bitcoinj-core')
+}
+
+tasks.withType(JavaCompile) {
+    options.compilerArgs.addAll(['--release', '8'])
+    options.compilerArgs << '-Xlint:deprecation'
+    options.encoding = 'UTF-8'
+}
+
+ext.moduleName = 'org.bitcoinj.test-support'
+
+jar {
+    inputs.property("moduleName", moduleName)
+    manifest {
+        attributes 'Automatic-Module-Name': moduleName
+    }
+}
+
+def minGradleArchiveClassifierVersion = GradleVersion.version("5.0")
+
+task javadocJar(type: Jar, dependsOn: javadoc) {
+    if (GradleVersion.current().compareTo(minGradleArchiveClassifierVersion) > 0) {
+        archiveClassifier.set('javadoc')
+    } else {
+        classifier = 'javadoc'
+    }
+    from javadoc.destinationDir
+}
+
+task sourcesJar(type: Jar, dependsOn: classes) {
+    if (GradleVersion.current().compareTo(minGradleArchiveClassifierVersion) > 0) {
+        archiveClassifier.set('sources')
+    } else {
+        classifier = 'sources'
+    }
+    from sourceSets.main.allSource
+}
+
+artifacts {
+    archives sourcesJar
+    archives javadocJar
+}
+
+publishing {
+    publications {
+        mavenJava(MavenPublication) {
+            artifactId = 'bitcoinj-test-support'
+            from components.java
+            artifact sourcesJar
+            artifact javadocJar
+
+            pom {
+                description = 'A Java Bitcoin library'
+            }
+        }
+    }
+}
```

Comment:
I would think this is all not needed.

Reply:
Sorry, the `dependencies` section is indeed needed.

Reply:
I'm talking about lines 20-67 that I think are not needed.

Reply:
I would leave the automatic module name just to reserve the name. It may be useful of we start running tests in modular mode.

But the rest can be removed since we're not publishing.